src/modules/db_operations/operations.py

```diff
@@ -58,24 +56,29 @@ def drop_and_populate_strains(app, db):
   drop_tables(app, db, tables=[Strain.__table__])
   load_strains(db)
 
+
 def drop_and_populate_wormbase_genes(app, db, wb_ver: str):
   logger.info(f"Running Drop and Populate wormbase genes with version: {wb_ver}")
-  gene_gff_fname = download_external_db('GENE_GFF_URL', wb_ver=wb_ver)
-  gene_gtf_fname = download_external_db('GENE_GTF_URL', wb_ver=wb_ver)
-  gene_ids_fname = download_external_db('GENE_IDS_URL', wb_ver=wb_ver)
-  homologene_fname = download_external_db('HOMOLOGENE_URL')
-  ortholog_fname = download_external_db('ORTHOLOG_URL')
-  
+
+  file_manager = DatasetManager(wb_ver=wb_ver)
```

Comment:
not an issue, but semantically speaking, consider renaming `file_manager` to `dataset_manager`

Reply:
Fixed in [68758cb](https://github.com/AndersenLab/CAENDR/pull/131/commits/68758cbb0d2d6b9d02ac4c68842bd5ce0cfca3ac)